src/elements/table/Table.js

```diff
@@ -259,8 +259,8 @@ export class NovoTable {
      * @param filter
      */
     onFilterClick(column, filter) {
-        if (Array.isArray(column.filter)) {
-            if (column.filter.includes(filter)) {
+        if (Array.isArray(column.filter) && column.multiple) {
```

Comment:
Does this break the other table filters that use select boxes and were multiple by default?